lte/cloud/go/services/lte/obsidian/handlers/handlers_test.go

```diff
@@ -644,6 +744,45 @@ func TestCellularPartialUpdate(t *testing.T) {
 	expected.Version = 3
 	assert.Equal(t, expected, actualN2)
 
+	// Fail to put ngc config to a network without cellular network configs
+	ngcConfig := &lteModels.NetworkNgcConfigs{SuciProfiles: []*lteModels.SuciProfile{
```

Comment:
Similar to above comment: let's go ahead and fully format this, by moving suciprofiles to its own line

Reply:
done